zerver/actions/realm_settings.py

```diff
@@ -486,3 +487,57 @@ def do_send_realm_reactivation_email(realm: Realm, *, acting_user: Optional[User
         language=language,
         context=context,
     )
+
+
+def do_send_realm_deactivation_email(realm: Realm, acting_user: Optional[UserProfile]) -> None:
+    # if realm is not deactivated, but changed the domain
+    if not acting_user:
+        return
```

Comment:
Can you explain this comment?

Reply:
It is check, to not send emails to owners if there is a change in the realm subdomain.
Because the process of changing realm subdomain, we deactivate realm subdomain in which it is hosted and then, make another realm with other subdomain.

Reply:
OK, this should be controlled by a new explicit flag, rather than piggy-backing on the fact that one code path does something slightly different. 

@alya product-wise, I guess we should decide what we want to do for the emails when an organization was deactivated on the command-line by a server administrator either in the command line or via the administrator /activity/support panel; probably just needs an extra option or two in the template, e.g. "Your organization was deactivated by {Zulip Cloud Support}" or something. We probably don't want to have this accidentally send the name of the support/security team member who clicked the button to deactivate an organization.